addons/hr_timesheet/views/project_views.xml

```diff
@@ -306,6 +306,7 @@
                 </xpath>
                 <xpath expr="//div[hasclass('o_project_kanban_boxes')]" position="after">
                     <t t-set="badgeColor" t-value="'border-success'"/>
+                    <t t-set="badgeColor" t-value="'border-warning'" t-if="record.remaining_hours.raw_value/record.allocated_hours.raw_value &lt; 0.2 and record.remaining_hours.raw_value/record.allocated_hours.raw_value &gt; 0"/>
                     <t t-set="badgeColor" t-value="'border-danger'" t-if="record.remaining_hours.raw_value &lt; 0"/>
```

Comment:
`t-elif` instead of `t-if` ? 🤔 🙂 
since it is related to the line you added above. 🙂